src/Uno.UI/UI/Xaml/Controls/PasswordBox/PasswordBox.cs

```diff
@@ -130,5 +133,43 @@ public override string GetAccessibilityInnerText()
 			// We don't want to reveal the password
 			return null; 
 		}
+
+		public bool IsPasswordRevealButtonEnabled
+		{
+			get => (bool)this.GetValue(IsPasswordRevealButtonEnabledProperty);
+			set
+			{
+				this.SetValue(IsPasswordRevealButtonEnabledProperty, value);
+			}
+		}
+
+		public static readonly DependencyProperty IsPasswordRevealButtonEnabledProperty =
+			DependencyProperty.Register(
+				nameof(IsPasswordRevealButtonEnabled),
+				typeof(bool),
+				typeof(PasswordBox),
+				new PropertyMetadata(
+					defaultValue: true,
+					propertyChangedCallback: (s, e) => ((PasswordBox)s)?.OnIsPasswordRevealButtonEnabledChanged(e)
+				)
+			);
+
+		private void OnIsPasswordRevealButtonEnabledChanged(DependencyPropertyChangedEventArgs e)
+		{
+			_isButtonEnabled = IsPasswordRevealButtonEnabled;
+			UpdateButtonStates();
+		}
+
+		private void UpdateButtonStates()
```

Comment:
It's kinda confusing that this duplicates the functionality of a different method with the same name in `TextBox`, with no sync-up between the two

Reply:
Removed method since it was not being used anywhere else and added functionality to the caller.